addons/binding/org.openhab.binding.omnilink/ESH-INF/thing/unit.xml

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="omnilink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<!-- These are all objects that are types of Units -->
+	
+	<thing-type id="unit">
```

Comment:
So this is a raw unit thing type that should work regardless of upb/x10/flag/etc...